bin/deb_local

```diff
@@ -187,15 +188,19 @@ end
 
 o_selected_packages = options.parse(ARGV)
 
+Autoproj::Packaging.root_dir = Autoproj.root_dir
+
 build_options[:release_name] ||= 'local'
 if File.exists? o_patch_dir
     build_options[:patch_dir] = o_patch_dir
 else
-    Autoproj.warn "Patch directory: #{patch_dir} does not exist"
+    Autoproj::Packaging.warn "Patch directory: #{patch_dir} does not exist"
     exit 0
 end
 
 packager = Autoproj::Packaging::Debian.new(build_options)
+autoprojadaptor = Autoproj::Packaging::Autoproj1Adaptor.new(build_options)
```

Comment:
Do you want me to add this to this pull request, i.E. remove dependencies on autoproj from deb_local, deb_package, lib/rock/packaging/jenkins etc as well?